### PR TITLE
add podspec to support react-native 0.60+ autolinking on iOS

### PR DESCRIPTION
**Pull request by MattiasMansson**

### CODE REVIEW

**Comment by MattiasMansson** [comment]
Think it's better to have it merged into your repository than having fork of a fork :)

File: react-native-share-extension.podspec [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  s.name         = package['name']
+  s.version      = package['version']
+  s.summary      = package['description']
+  s.license      = package['license']
+
+  s.authors      = package['author']
+  s.homepage     = package['homepage']
+  s.platform     = :ios, "9.0"
+
+  s.source       = { :git => "https://github.com/MattiasMansson/react-native-share-extension", :tag => "v#{s.version}" }
```
Comment (frenberg) [inline react-native-share-extension.podspec]: Do you know what will happen here if you decided to remove your fork?

Comment (frenberg) [reply react-native-share-extension.podspec]: or for that matter - will a pod install acctually fetch your code instead?

Comment (MattiasMansson) [reply react-native-share-extension.podspec]: Oh right, I need to update it to match your repository

**Pull request closed without merging**